frontend/src/layout/navigation/TopNavigation.tsx

```diff
@@ -251,6 +282,10 @@ export function TopNavigation(): JSX.Element {
                         }
                     )
                 }
+                style={{
+                    cursor: isProjectCreationForbidden ? 'default' : undefined,
+                    color: isProjectCreationForbidden ? 'var(--text-muted)' : undefined,
```

Comment:
```suggestion
                    color: isProjectCreationForbidden ? 'var(--text-muted)' : 'var(--primary)',
```

Reply:
Would prefer not to, to keep consistency between "Create new project" and "Create new organization"

Reply:
I don't think we need consistency among these, they're in different places, plus adding a project is probably a lot more frequent than an org. I'm also wondering if the create org should also be primary color.

Reply:
Alrighty, made both primary